README.md

```diff
@@ -68,9 +68,9 @@ environment variables
   In addition, if ``SETUP_CMD`` contains the following flags, extra dependencies are installed:
 
     * ``--coverage``: the coverage and coveralls packages are installed
+    * ``--cov``: the pytest-cov and coveralls packages is installed
```

Comment:
is -> are

Reply:
Oh. I would rather fix this typo in the other PR if that's OK, so we don't have to rerun CI for this once again.

Reply:
fixed in #244